### PR TITLE
add column in pos window

### PR DESCRIPTION
**Pull request by Andojeda**
## Feature

A column was added in the table of the point of sale window to show the currency conversion and the language translation was modified

convertir

### CODE REVIEW

File: src/components/ADempiere/Form/VPOS/Order/orderLineMixin.js [hunk 1]
```diff
@@ -50,6 +50,12 @@ export default {
           isNumeric: true,
           size: '110px'
         },
+        convertamount: {
+          columnName: 'GrandTotal',
+          label: this.$t('form.pos.collect.convertAmount'),
```
Comment (yamelsenih) [inline src/components/ADempiere/Form/VPOS/Order/orderLineMixin.js]: Please change this for:
```Javascript
this.$t('form.pos.collect.convertAmount')
```

File: src/lang/ADempiere/es.js [hunk 2]
```diff
@@ -444,7 +444,7 @@ export default {
         pending: 'Pendiente',
         payment: 'Pago',
         change: 'Cambio',
-        convertAmount: 'Convertir Cantidad',
+        convertAmount: 'Convertir Monto',
```
Comment (yamelsenih) [inline src/lang/ADempiere/es.js]: Change this for convertedAmount

File: src/components/ADempiere/Form/VPOS/Order/orderLineMixin.js [hunk 3]
```diff
@@ -50,6 +50,12 @@ export default {
           isNumeric: true,
           size: '110px'
         },
+        convertamount: {
```
Comment (yamelsenih) [inline src/components/ADempiere/Form/VPOS/Order/orderLineMixin.js]: Change their to convertedAmount

File: src/components/ADempiere/Form/VPOS/Order/orderLineMixin.js (hunk 1 above)
Comment (yamelsenih) [inline src/components/ADempiere/Form/VPOS/Order/orderLineMixin.js]: Change this to convertedAmount instead

File: src/lang/ADempiere/es.js (hunk 2 above)
Comment (yamelsenih) [inline src/lang/ADempiere/es.js]: Please change translation to: **Monto Convertido**

**Review by yamelsenih — CHANGES_REQUESTED** [review]

**Review by Andojeda — COMMENTED** [review]
what was required was modified

**Comment by oscarinak** [comment]
The PR revision was carried out, and it contemplates the new change of the converted column, however it does not show the converted amounts, if the point of sale "USD" is entered in that column it should show the amount in bolivars, and if enter in the point of sale "VES" you must convert that price into dollars

image

image

**Comment by oscarinak** [comment]
bad

File: src/components/ADempiere/Form/VPOS/Order/orderLineMixin.js [hunk 4]
```diff
@@ -50,6 +50,12 @@ export default {
           isNumeric: true,
           size: '110px'
         },
+        convertedAmount: {
+          columnName: 'GrandTotal',
```
Comment (yamelsenih) [inline src/components/ADempiere/Form/VPOS/Order/orderLineMixin.js:54]: Change grandtotal to convertedAmount

**Review by yamelsenih — CHANGES_REQUESTED** [review]

**Pull request closed without merging**